packages/wagmi/src/components/user-profile/DefaultView.tsx

```diff
@@ -32,11 +35,18 @@ export const DefaultView: FC<DefaultProps> = ({
   setView,
 }) => {
   const { disconnect } = useDisconnect()
-  const { data: price } = usePrice({
+  const { data: _price } = usePrice({
     chainId,
     address: Native.onChain(chainId).wrapped.address,
+    enabled: isWNativeSupported(chainId),
   })
 
+  const price = useMemo(() => {
```

Comment:
I see this done in multiple places now, might be nice to have a `usePriceFraction` hook or something? Just a thought though, don't worry about it